contrib/docker/.env.prod.template

```diff
@@ -247,14 +248,13 @@ SECURITY_EMAIL=
 
 # MAP EDITOR SETTINGS
 ENABLE_MAP_EDITOR=true
-# Enable broadcast areas in map editor (beta)
+# Enable broadcast areas in map editor
```

Comment:
This is still pretty much beta.

We have had a number of issues and are not 100% sure this is stable and scales well. For instance, it's not yet enable in our cloud offering.

```suggestion
# Enable broadcast areas in map editor (beta)
```

Reply:
Ah, that explains a lot. I just assumed, it must be well tested by now and attributed issues with it to my inability to setup stuff correctly. ^^